Library Management Website

Nitish36 opened the pull request:
Created the website using html, css, javascript and jquery. Applied a scroll feature which will scroll the list of books and also added a search section wherein the user can search the particular book by author, date, book name or title. Please merge this with the original work

Thank You :)

Kushal997-das commented:
@Nitish36 Mention your Issue Number

Nitish36 commented:
My issue number is #915

Kushal997-das commented:
@Nitish36 Hi check my contributing guidelines and do the work accordingly, as I can see you do not have a proper `readme. md` , describe your project so that others can understand, what tech stack you used and deploy your project so that others can see that.

Nitish36 commented:
Hey I have updated the readme as you have told and added explanation of each output screenshot as well as the tech stacks used. Do merge it with your repo

Thank you :)

Nitish36 commented:
Hey I have made all the required changes as you have told. If there anymore do tell. Do merge it with your repo.

Thank You :)

Kushal997-das reviewed (changes requested):
@Nitish36 I checked your deploy link and I searched the title, author, etc. like you mentioned in the readme.md file  but it was not working
